tools/pot/README.md

```diff
@@ -20,50 +20,72 @@
 and run on CPU with the OpenVINO&trade;.
-* A representative calibration dataset representing a use case scenario, for example, 300 images.
+* A representative calibration dataset representing a use case scenario, e.g. 300 images. 
+
+Figure below shows the optimization workflow:
+![](docs/images/workflow_simple.png) 
 
 Post-training Optimization Tool provides the following key
 features:
 
 * Two post-training 8-bit quantization algorithms: fast [DefaultQuantization](openvino/tools/pot/algorithms/quantization/default/README.md) and precise [AccuracyAwareQuantization](openvino/tools/pot/algorithms/quantization/accuracy_aware/README.md).
 * Compression for different hardware targets such as CPU and GPU.
 * Multiple domains: Computer Vision, Natural Language Processing, Recommendation Systems, Speech Recognition.
+* [Command-line tool](docs/CLI.md) that provides a simple interface for basic use cases.
 * [API](openvino/tools/pot/api/README.md) that helps to apply optimization methods within a custom inference script written with OpenVINO Python* API.
-* Symmetric and asymmetric quantization schemes. For details, see the [Quantization](openvino/tools/pot/algorithms/quantization/README.md) section.
-* Per-channel quantization for Convolutional and Fully-Connected layers.
+* (Experimental) [Ranger algorithm](@ref pot_ranger_README) for model prodection in safity critical cases.
 
 The tool is aimed to fully automate the model transformation process without a need to change the model on the user's side. For details about 
 the low-precision flow in OpenVINO&trade;, see the [Low Precision Optimization Guide](docs/LowPrecisionOptimizationGuide.md).
 
 For benchmarking results collected for the models optimized with POT tool, see [INT8 vs FP32 Comparison on Select Networks and Platforms](@ref openvino_docs_performance_int8_vs_fp32).
 
-POT is opensourced on GitHub as a part of [https://github.com/openvinotoolkit/openvino](https://github.com/openvinotoolkit/openvino).
+POT is opensourced on GitHub as a part of OpenVINO and available at https://github.com/openvinotoolkit/openvino/tools/pot.
 
 Further documentation presumes that you are familiar with the basic Deep Learning concepts, such as model inference,
 dataset preparation, model optimization, as well as with the OpenVINO&trade; toolkit and its components such 
 as  [Model Optimizer](@ref openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide) 
 and [Accuracy Checker Tool](@ref omz_tools_accuracy_checker).
 
-## Use POT
-![](docs/images/workflow.png) 
+## Usage options
+![](docs/images/use_cases.png) 
+
+The POT provides three basic usage options:
+* **Command-line interface (CLI)**:
+  * [**Simplified mode**](@ref pot_docs_simplified_mode): this option can be used if the model from Computer Vision domain and there is an unannotated dataset that can be used for optimization. Please note that there can be deviation of accuracy after optimization using this method.
```

Comment:
Maybe we could describe it as expandable description for each option. @tsavina , do you know if there is such option when the reader clicks on the option and expand the detailed description?

Reply:
I think this information should be stated in the corresponding description file for this mode. Moreover, once we support the load from binary data, which I really want to have, this information will be invalid. I am aginst of overloading the main README.

Reply:
Ok, let's have it in dedicated pages but I would still clearly state what is needed for each option to avoid the situation when the reader will need to read 3 other documents to understand which flow can be used in their particular case.

Reply:
I believe that the prerequisites are shown in the picture above and the user should select the best option based on the availability of model, data, code and AC config. If it is not clear I would suggest revising this in a different PR and involving the documentation team. BTW, @tsavina will change the images in the next PRs.